tests/model_serving/model_runtime/conftest.py

```
@@ -1,9 +1,49 @@
+def rhoai_version(admin_client: DynamicClient) -> Version | None:
+    """Detect RHOAI product version from env var or cluster CSV."""
+    env_version = get_rhoai_version_from_env()
+    if env_version:
+        LOGGER.info(f"RHOAI version from environment: {env_version}")
+        return env_version
+
+    try:
+        cluster_version = get_product_version(admin_client=admin_client)
+        LOGGER.info(f"RHOAI version from cluster CSV: {cluster_version}")
+        return cluster_version
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)